test/e2e/scale_test.go

```diff
@@ -46,7 +46,6 @@ func testScaleCluster(t *testing.T) {
 	g.Expect(err).NotTo(HaveOccurred())
 
 	// Scale up Pravega cluster, increase bookies and segment store size by 1
-	pravega.Spec.Bookkeeper.Replicas = 4
```

Comment:
How are we going to perform test when this change is merged? Are we going to "mock" Bookies?

Reply:
For unit tests there is BK is not needed. For E2E tests we would do the same things we do for ZK.